### PR TITLE
Add  testing framework bats and fix #4

### LINKED ISSUE #4: PR comment behavior
When "yamllint_comment" is set to false the action is still comment to the PR.

### PR DESCRIPTION
**Pull request by steinbrueckri**
# TODOs
- [x] keep both 1 or true supported for INPUT_YAMLLINT_COMMENT
- [x] fix, If yamllint passes & comment is set to true, it will still comment and empty result (same behavior as now)
- [x] update readme because of the tests

### CODE REVIEW

File: src/entrypoint.sh [hunk 1]
```diff
@@ -28,7 +28,7 @@ function parse_inputs {
     fi
 
     yamllint_comment=0
-    if [ "${INPUT_YAMLLINT_COMMENT}" != "0" ] || [ "${INPUT_YAMLLINT_COMMENT}" != "false" ]; then
+    if [ "${INPUT_YAMLLINT_COMMENT}" == "true" ]; then
```
Comment (karancode) [inline src/entrypoint.sh:31]: I would like to keep both `1` or `true` supported.

**Comment by karancode** [comment]
First of all thank you so much for all these changes and add-ons 🙇

**Comment by karancode** [comment]
Thanks again for all the changes and updates.
It would be great if you could add a little readme about the `tests` part and the GITHUB_EVENT.json...
Overall LGTM 🙇

**Comment by steinbrueckri** [comment]
@karancode Maybee, I remove the GITHUB_EVENT.json. It was just an idea to test the curl on the basis of a prepared JSON file that looks like an event.

**Comment by karancode** [comment]
@steinbrueckri awesome. Once removed will merge and cut a new release tag! 🥳

**Comment by steinbrueckri** [comment]
@karancode READY! 🙌 🎉

**Pull request merged**